modules/machineset-aws-existing-placement-group.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-machinesets/creating-machineset-aws.adoc
+// * machine_management/control_plane_machine_management/cpmso-using.adoc
+
+ifeval::["{context}" == "cpmso-using"]
+:cpmso:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="machineset-aws-existing-placement-group_{context}"]
+= Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets
```

Comment:
I don't think that only EFA instances are supported. This should be like all the instances? @JoelSpeed

Reply:
EFA is an optional feature that _can_ be used with placement groups, it isn't required though.

EFA is much more effective at having low latency between hosts when used with a correctly configured placement group

Reply:
I tired to get at that with:
> EFA instances do not require placement groups, and you can use placement groups for purposes other than configuring an EFA. This example uses both to demonstrate a configuration that can improve network performance for machines within the specified placement group.

basically, trying to show a likely use case in docs but not making the claim that _only_ this use case is supported. Do y'all think what I have is clear enough for that?

Reply:
As we are taking this part of docs as an example to use the placementGroups, I am good with that.